### DISCUSSION TITLE
WiX4 - Cannot find torch.exe in WiX 4

### DISCUSSION THREAD

**Discussion by oleg-shilo** [wixtoolset/issues#7649] [Questions] [answered]
It looks like torch.exe is no longer available in WiX4. Unless I just missed it.
What is the recommended alternative?

**Comment by BMurri** [comment, marked as answer]
I assume `wix msi transform` but I don't really know for sure.

**Reply by oleg-shilo** [reply]
Thank you, will try

**Comment by oleg-shilo** [comment]
I can confirm now that in WiX4 `wix msi transform` works exactly as `torch` in WiX3.
